pytorch_lightning/loops/optimization/manual_loop.py

```diff
@@ -146,6 +146,8 @@ def on_run_end(self) -> _OUTPUTS_TYPE:
 
     def _on_before_step(self) -> None:
         self.optim_step_progress.increment_ready()
+        self.trainer.profiler.start("optimizer_step")
```

Comment:
In automatic optimization, we profile the `optimizer_step` already by calling `_call_lightning_module_hook("optimizer_step")`

And in manual optimization, as shown here

Reply:
> This means the profiler entry was duplicated too.

does that warrant a bugfix changelog entry? :)

Reply:
I wouldn't say so. There are no guarantees over the profiler summary format so I wouldn't call it a bugfix. It's a change